src/databricks/labs/ucx/install.py

```diff
@@ -120,6 +121,9 @@ def deploy_schema(sql_backend: SqlBackend, inventory_schema: str):
             functools.partial(table, "udfs", Udf),
             functools.partial(table, "logs", LogRecord),
             functools.partial(table, "recon_results", ReconResult),
+            functools.partial(
```

Comment:
Adding `direct_file_system_access_in_queries` table will be done in PR https://github.com/databrickslabs/ucx/pull/2599